src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/QuestionSteps.java

```diff
@@ -261,4 +274,17 @@ public void the_response_contains_n_questions(int count) throws Throwable {
         AllQuestionsResponse questionResponses = mapper.readValue(rawJson, AllQuestionsResponse.class);
         assertEquals(count, questionResponses.getQuestions().size());
     }
+
+    @And("^the question id matches$")
```

Comment:
This is to check that the returned Question ID is the same as the one requested in the GET request